app/code/Magento/InventoryApi/Api/SourceRepositoryInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryApi\Api\Data\SourceSearchResultsInterface;
+
+/**
+ * @api
+ */
+interface SourceRepositoryInterface
+{
+    /**
+     * Save Source data.
+     *
+     * @param SourceInterface $source
+     * @return SourceInterface
+     *
+     * @throws CouldNotSaveException
+     */
+    public function save(SourceInterface $source);
+
+    /**
+     * Load Source data by given sourceId.
+     *
+     * @param int $sourceId
+     * @return SourceInterface
+     * @throws NoSuchEntityException
+     */
+    public function get($sourceId);
+
```

Comment:
please provide a description why there is no delete method in this repository.

Because community guys would be curious about this, as having delete method a general practice for magento repositories